models/public/semantic-segmentation-hrnet-v2-c1/semantic-segmentation-hrnet-v2-c1.md

```diff
@@ -0,0 +1,110 @@
+# semantic-segmentation-hrnet-v2-c1
```

Comment:
Why did you change name of the model? The `hrnet-v2-c1` was pretty good.

Reply:
It was my comment to change, but I can not find it now.
We have potential request for adding hrnet pose estimation too

Reply:
my opinion, better <model_name>-<task>
```suggestion
# hrnet-v2-c1-segmentation
```